scripts/utils.sh

```diff
@@ -57,11 +57,7 @@ run_csi_proxy_integration_tests() {
   "& {
     $ErrorActionPreference = \"Stop\";
     Import-Module (Resolve-Path(\"utils.psm1\"));
-    Run-CSIProxyIntegrationTests -test_args \"--test.v --test.run TestAPIGroups\";
-    Run-CSIProxyIntegrationTests -test_args \"--test.v --test.run TestFilesystemAPIGroup\";
-    Run-CSIProxyIntegrationTests -test_args \"--test.v --test.run TestDiskAPIGroup\";
-    Run-CSIProxyIntegrationTests -test_args \"--test.v --test.run TestVolumeAPIs\";
-    Run-CSIProxyIntegrationTests -test_args \"--test.v --test.run TestSmbAPIGroup\";
+    Run-CSIProxyIntegrationTests -test_args \"--test.v\";
```

Comment:
I think I did this so that I'd get output for each group independently so that one test failure doesn't skip running the other tests

Reply:
Ah. This is temporary as we migrate since we have two sets of tests, and it's kind of painful to have to keep adding new lines here to do the v2 tests.